### ISSUE TITLE
[ui, bug] Can Write Variable check is not permissive enough

### ISSUE THREAD

**Issue by philrenaud** [hashicorp/nomad#14061]
Two bugs:

- The link to create a new variable in the "no variables found" form is active even when the button is disabled 
![image](https://user-images.githubusercontent.com/713991/183720170-f911ac6e-340a-4926-bcf9-f259fad65066.png)

- A policy with `write` in one of the paths should have access to create a new variable from root; because if there are no yet-created variables, how would the user even be able to create one?

```
    # full access to secrets in all project paths
    path "project/*" {
      capabilities = ["write", "read", "destroy", "list"]
    }

    # full access to secrets in all project paths
    path "*" {
      capabilities = ["list"]
    }
```

**Comment by ChaiWithJai** [comment]
> A policy with write in one of the paths should have access to create a new variable from root; because if there are no yet-created variables, how would the user even be able to create one?

I don't think this should block `1.4.0`. This user journey should get a design and specifically address how we'll address error state before we pick up work on this IMO.

**Comment by philrenaud** [comment]
@ChaiWithJai we don't block users from root variable creation at the API or CLI levels and we shouldn't block them at the UI level either

**Comment by philrenaud** [comment]
Resolved by #14067
